### ISSUE TITLE
Add release tags for versions of PyCubed/Battery Board

### ISSUE THREAD

**Issue by polygnomial** [pycubed/hardware#6]
# Rationale
We realized we'd been using a slightly older version of v04 with some slight differences from most of the documentation. It would make life a lot easier to have a single source of truth release tag so that it is clear exactly which version is in use. This could also be extended beyond v1/2/4 to subversions.

**Comment by polygnomial** [comment]
I'd be happy to submit a PR, though I'm not entirely sure how/whether release tags work in PRs.

**Comment by maholli** [comment]
Yes, absolutely! I think SSI had access to v04 as it was in development and that's why you ended up with a different rev.

When I roll out v5 (coming soon!!) I'll make sure to tag it with the appropriate release.

**Comment by polygnomial** [comment]
😊🛰
